### PR TITLE
Arguments

### PR DESCRIPTION
**Pull request by jberger**
This is the successor request to #2 which can be merged once we figure out why the test stalls.

### CODE REVIEW

**Comment by plicease** [comment]
rebased onto master: https://github.com/plicease/Mojo-Phantom/tree/arguments

**Comment by Eckankar** [comment]
I've taken a look at this again with a colleague, and we have no idea what could be causing your PhantomJS to hang, if it doesn't on master as well. (In which case, #2 outlines something that can cause PhantomJS to hang.)

(We even considered if spaces in the temporary filename could cause issues, but we concluded that it doesn't appear to.)

An alternate route could be to mock it out, and not actually call PhantomJS, but I'm not quite sure what could be meaningfully mocked out to test that this feature works, without making the test trivial.

In short, I don't quite know how to proceed. :)

Which setup and PhantomJS version did you experience this with?

**Comment by jberger** [comment]
Hmmmm, so deleting the branch (on my github remote) closes the PR from your remote?! I'm very confused.

**Comment by jberger** [comment]
Oh right, because **I** opened this PR after your branch caused test failures ... :s
